src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -26,22 +26,41 @@ namespace Microsoft.Identity.Client.Http
     /// </remarks>
     internal class HttpManager : IHttpManager
     {
+        // referenced in unit tests, cannot be private
+        public const int DEFAULT_ESTS_MAX_RETRIES = 1;
+        // this will be overridden in the unit tests so that they run faster
+        public static int DEFAULT_ESTS_RETRY_DELAY_MS { get; set; } = 1000;
+
         protected readonly IMsalHttpClientFactory _httpClientFactory;
-        private readonly IRetryPolicy _retryPolicy;
+        private readonly bool _isManagedIdentity;
+        private readonly bool _withRetry;
         public long LastRequestDurationInMs { get; private set; }
 
         /// <summary>
-        /// A new instance of the HTTP manager with a retry *condition*. The retry policy hardcodes: 
-        /// - the number of retries (1)
-        /// - the delay between retries (1 second)
+        /// Initializes a new instance of the <see cref="HttpManager"/> class.
         /// </summary>
+        /// <param name="httpClientFactory">
+        /// An instance of <see cref="IMsalHttpClientFactory"/> used to create and manage <see cref="HttpClient"/> instances.
+        /// This factory ensures proper reuse of <see cref="HttpClient"/> to avoid socket exhaustion.
+        /// </param>
+        /// <param name="isManagedIdentity">
+        /// A boolean flag indicating whether the HTTP manager is being used in a managed identity context.
+        /// </param>
+        /// <param name="withRetry">
+        /// A boolean flag indicating whether the HTTP manager should enable retry logic for transient failures.
+        /// </param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown when <paramref name="httpClientFactory"/> is null.
+        /// </exception>
         public HttpManager(
             IMsalHttpClientFactory httpClientFactory,
-            IRetryPolicy retryPolicy)
+            bool isManagedIdentity,
+            bool withRetry)
```

Comment:
rename to `isRetryEnabled`

Reply:
The alternative here is to go to all references for SendRequestAsync and manually pass in the default retry policy. I think it makes more sense to define it here, once, if no retryPolicy was provided.

Currently in .NET, all STS and MI requests have the default LinearRetryPolicy.